docs/common/project/how-to-release.md

```diff
@@ -255,3 +260,63 @@ Java artifacts are available from Maven Central.
 
 Thanks to everyone for contributing!
 ```
+
+### Documentation Release
+
+Documentation needs to be updated as a part of Iceberg release after a release candidate is passed.
+The commands described below assume the `iceberg-docs` repository and `iceberg` repository are in the same parent directory locally, 
+and the release manager is executing commands in the `iceberg` repository.
+Adjust the commands accordingly if it is not the case.
+
+#### Release notes
+
+Release notes need to be drafted and published in the `iceberg` repository and reviewed by the community.
+
+#### Iceberg version update
+
+Before release, publish a PR in the `iceberg-docs` repository to update `landing-page/config.toml` and `docs/config.toml` to use the new Iceberg version number.
+
+#### Documentation update
+
+To start the release process, run the following steps in the `iceberg-docs` repository to copy docs over:
+
+```shell
+rm -rf ../iceberg-docs/docs/content/docs
+rm -rf ../iceberg-docs/landing-page/content/common
+cp -r docs/versioned ../iceberg-docs/docs/content/docs
+cp -r docs/common ../iceberg-docs/landing-page/content/common
```

Comment:
Is this going to be the case if most pages are maintained in iceberg-docs? Maybe we don't delete common, but copy the spec.md file from iceberg to iceberg-docs.

Reply:
If we just have `spec.md` in `common`, we just remove that specific file and copy that file instead of the entire `common` folder.